src/helpers/useUndo.ts

```diff
@@ -178,7 +178,7 @@ export const useUndo = ({
       description: 'Undo changes',
       callback: (event) => {
         event.preventDefault();
-        if (!disabled) {
+        if (!disabled && canUndo) {
```

Comment:
@amcdnl Not sure about that, please confirm.

Reply:
Good catch!